### PR TITLE
Bugfix/mainpage background

### LINKED ISSUE #388: Fix main page background resolution
https://devadventcalendar.pl/
image
There should be visible Santa but is not (on Chrome)

### PR DESCRIPTION
**Pull request by BlackMorzan**
Changed padding in style.css so in max resolution picture should fit in visible area.

## Description
Picture was upscaled in y axis to meet size of vertical padding of 395px, text and counter. That also make picture bigger in x axis and enter invisible space on the right. I guess verticall padding anticipated smaller couter, or none at all.

## Motivation and context
It should make Santa visible - fix to closes #388 

## How has this been tested?
I just eyeballed it for now.

## Screenshots (if appropriate)
Bez tytułu
FB_Button
Bez tytułu
Santa 1024
Santa 1336
Santa 1440

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by WTobor** [comment]
- why have you deleted AdminServiceTest.cs?
- is that solution with size change working on the phone and PC?

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css [hunk 1]
```diff
@@ -2064,8 +2064,8 @@ footer{
 	background-size: cover;
 	background-repeat: no-repeat;
 	overflow: hidden;
```
Comment (BlackMorzan) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2066]: This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css [hunk 2]
```diff
@@ -2064,8 +2064,8 @@ footer{
...
-	padding-top: 205px;
-	padding-bottom: 190px;
+	padding-top: 130px;
+	padding-bottom: 140px;
```
Comment (BlackMorzan) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)

**Comment by BlackMorzan** [comment]
> * is that solution with size change working on the phone and PC?
Deletion of AdminServiceTest.cs seems related to merge with develop 7ebbe9d. I might started branch from wrong place.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css (hunk 2 above)
Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: I am unsure how this site was intended to react to changing resolution therefore I don't know the right approach to take while debugging.

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: you can show screenshots how does it look like on different resolution in a browser (F12 and Toggle device toolbar)

Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: Added screenshots of "mobile device mode" in screenshots section

Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: Ok. Different approach is needed

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:2068]: screenshots look ok - maybe not perfect, but ok ;)

Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css]: I wasn't happy how it looked on devices over 1200px wide. It required a bit of learning of css but now it looks ok :)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/css/responsive.css [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+    }
+}
+
+@media screen and (min-width: 1200px) and (max-width: 1999px) {
+    .countdown_section {
+        padding-bottom: 150px;
+        padding-top: 150px;
+    }
+}
+
```
Comment (BlackMorzan) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/css/responsive.css:14]: this chunk of code changes padding to make Santa visible on HD, fullHD, and 4K monitors - I think it is done for now (although 4K could use more changes to look good)

**Comment by BlackMorzan** [comment]
I checked AdminServiceTest. It looks like it have been refactored and moved to IntegrationTests folder. I also added screenshots for 1024p, 1336p, and 1440p. I'd say it looks fine.

**Review by WTobor — APPROVED** [review]

**Comment by WTobor** [comment]
@BlackMorzan are we ready to complete this PR? Do you have Hactoberfest account ready? :)

**Comment by BlackMorzan** [comment]
I'm ready! Let it by my 1'st Hactoberfest contribution :)

**Pull request merged**